### DISCUSSION TITLE
キースイッチのパッドをピンセットでショートでは動作するが、実際にキースイッチを差し込むと反応しないキーがある

### DISCUSSION THREAD

**Discussion by tamotamago** [Yowkees/keyball#360] [Q&A] [answered]
keyball 61 で全てのキースイッチのパッドをピンセットでショートさせて動作を確認できております。

![Screenshot 2023-09-26 at 0 37 37](https://github.com/Yowkees/keyball/assets/892431/c9ca41f5-7239-48b8-98d2-ec6eee9a4daf)

しかし、実際キースイッチをつけると多くのキーが反応しませんでした。

![Image 2023-09-25 at 23 53](https://github.com/Yowkees/keyball/assets/892431/8a1fa703-7cac-4e82-8589-dce2fdd49c1a)

試しに動作しているキーのキースイッチ一つを動作していない複数箇所に差し込み確認しましたが、変わらず動作しませんでした。

この場合はキーソケットのテンプラの可能性が高いでしょうか？

**Comment by tamotamago** [comment, marked as answer]
https://github.com/Yowkees/keyball/discussions/333#discussioncomment-6931616

投稿後、こちらのコメント拝見しまして、キーソケットのテンプラの可能性が大きいとのことなので、まずこちらから試します。
一度クローズいたします。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

おっしゃる通り、キーソケットのテンプラが濃厚です。
端子と基板の間にある隙間に注目します。ここから半田が染み出してくるのが見えれば、テンプラ無く確実に半田付け出来ています。

お手数ですがご確認お願いいたします。
